### ISSUE TITLE
Missing property shadowRoot

### ISSUE THREAD

**Issue by Pulpyyyy** [j-a-n/lovelace-wallpanel#30]
Hello @j-a-n 

I saw a new error message this morning but can't see any reference from the documentation.
![image](https://user-images.githubusercontent.com/32004297/188570532-2a1a10e7-3fe7-4e9b-ac49-f58a425b6f48.png)
Did i missed something ?
Regards

**Comment by j-a-n** [comment]
Are you using the latest version?
Does this error occur regularly?

**Comment by Vitani** [comment]
Getting the same in Firefox, freshly installed today. It was working, and now it's not after tweaking my config, but undoing my changes doesn't seem to have fixed it. Here's my config for reference:

```
wallpanel:
  enabled: true
  idle_time: 1
  screensaver_stop_navigation_path: /lovelace/Home
  image_url: https://unsplash.it/${width}/${height}?random=${timestamp}
  screensaver_entity: input_boolean.control_panel_screensaver
  info_animation:
    speed_x: 1
    speed_y: 0
  style:
    wallpanel-screensaver-info-box:
      '--wp-card-width': 450px
      background-color: '#00000099'
      box-shadow: >-
        0px 2px 1px -1px rgba(0, 0, 0, 0.2), 0px 1px 1px 0px rgba(0, 0, 0,
        0.14), 0px 1px 3px 0px rgba(0, 0, 0, 0.12)
    wallpanel-screensaver-info-box-content:
      zoom: 200%
      '--ha-card-background': none
      '--ha-card-box-shadow': none
      '--primary-background-color': '#111111'
      '--secondary-background-color': '#202020'
      '--primary-text-color': '#e1e1e1'
      '--secondary-text-color': '#9b9b9b'
  cards:
    - type: custom:digital-clock
      firstLineFormat: T
      secondLineFormat: ccc d MMM
```

**Comment by Vitani** [comment]
Update: I only get the above error when in edit mode of the dashboard, when in the "published" view, it works fine.

**Comment by j-a-n** [comment]
Thanks for the logs and config!
The problem should be fixed in version 3.4.
